conseil-api/src/test/scala/tech/cryptonomic/conseil/api/routes/platform/data/ethereum/EthereumDataOperationsTest.scala

```diff
@@ -249,5 +322,121 @@ object EthereumDataOperationsTest {
       transactionIndex = transaction3.transactionIndex
     )
     val logs: Seq[LogsRow] = List(log1, log2, log3)
+
+    val receipt1: ReceiptsRow = ReceiptsRow(
```

Comment:
Might be there is an easy way to factor out this repetition in a easy-to-understand smart-constructor method?
Or even use a single default object and use `.copy` to customize it in the test depending on what we need each time?

Reply:
I decided to use default object + copy. Mostly because in previous company I was working with object builders and there was 2x more code that we have now and we also had to test it properly. I believe, I managed to reduce the code and complexity.